KD_Lib/KA/PS.py

```diff
@@ -9,7 +9,7 @@
 from copy import deepcopy
 
 
-class KnowledgeAdjustment(BaseClass):
+class PS(BaseClass):
```

Comment:
Can the class name be a bit more meaningful? I mean i won't know which implementation is this classname for?

Reply:
Done